src/include/parser/alter_statement.h

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// alter_function_statement.h
+//
+// Identification: src/include/parser/alter_function_statement.h
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "parser/sql_statement.h"
+#include "common/sql_node_visitor.h"
+#include "parser/create_statement.h"
+
+namespace peloton {
+namespace parser {
+/**
+ * @class AlterTableStatement
+ * @brief Represents "ALTER TABLE add column COLUMN_NAME COLUMN_TYPE"
+ * TODO: add implementation of AlterTableStatement
```

Comment:
[trivial] maybe you forgot to remove todo...

Reply:
Thanks for pointing out!